src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
 
 function Resources({ getResourcesUrl }) {
   const [resources, setResources] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(null);
 
   useEffect(() => {
     axios
       .get(getResourcesUrl)
       .then(function(response) {
         // handle success
-        setResources(response.data.results);
+        setResources(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
   }, []);
```

Comment:
Ohh, interesting! I thought that an empty dependencies array was enough to make sure the code in the useEffect ran only once: 

```
If you want to run an effect and clean it up only once (on mount and unmount), you can pass an empty array ([]) as a second argument. This tells React that your effect doesn’t depend on any values from props or state, so it never needs to re-run. 
```

🤔 Maybe `resources` could be put in there? But in any case, this will be stripped out with React Query.